sdks/java/io/kafka/build.gradle

```diff
@@ -52,6 +52,7 @@ dependencies {
   provided library.java.jackson_dataformat_csv
   permitUnusedDeclared library.java.jackson_dataformat_csv
   implementation project(path: ":sdks:java:core", configuration: "shadow")
+  implementation project(path: ":runners:core-java")
```

Comment:
I would appreciate someone more familiar with desired deps structure to review this.
It might be preferrable to move the constants to somewhere in core out of runners

Reply:
cc @Abacn thoughts?

Reply:
Missed that. Yes kafka io should not depends on runners. This does break internal import